build/test.ps1

```diff
@@ -7,6 +7,9 @@ $ErrorActionPreference = 'Stop'
 $all_ok = $True
 Write-Host "Assembly version: $Env:ASSEMBLY_VERSION"
 
+Write-Host "##vso[task.logissue type=warning;]Forcing all tests to be skipped."
```

Comment:
This part of the change will need to be removed before this can be merged. We need to fix the Project Loader tests for this.

Reply:
Removed.